### ISSUE TITLE
tags command fail

### ISSUE THREAD

**Issue by luislavena** [vertiginous/pik#18]
Without too much documentation, assumed this:

  pik tag abc

And when executed:

  pik tags

Got this error:

<pre><code>
There was an error
 Error: private method `split' called for nil:NilClass

  in: pik/commands/tag_command.rb:11:in `execute'
  in: pik:35
</code></pre>

**Comment by vertiginous** [comment]
The tags command does need better documentation.   It's meant to be used in conjunction with the run, ruby, and gems commands, to allow you to run against a subset.  Look at features/tag_command.feature 

```
Scenario: run tagged versions
  When I run "pik 191" and "pik tag mingw",
  And I run "pik 186 383" and "pik tag mingw",
  And I run "pik tags mingw run echo."
  Then I should see "ruby 1.8.6 (2009-08-04 patchlevel 383) [i386-mingw32]"
  And I should see "ruby 1.9.1p243 (2009-07-16 revision 24175) [i386-mingw32]"
```

**Comment by vertiginous** [comment]
Fixed in 1dc818b36645b765778c93d92586ca145fe044a1
